packages/knip/test/plugins/tanstack-router.test.ts

```diff
@@ -11,14 +11,16 @@ test('Find dependencies with the tanstack-router plugin', async () => {
   const { issues, counters } = await main({
     ...baseArguments,
     cwd,
+    isProduction: true,
   });
 
   assert(issues.files.has(join(cwd, 'routes/-unused.ts')));
```

Comment:
Without `isProduction: true`, the test wasn't passing, `issues.files` was an empty `Set`. This makes me really unsure my fix is correct...